src/debugTransform.ts

```diff
@@ -29,7 +29,7 @@ export default declare(api => {
         if (!isAwaitExpression && !isCallExpression)
           return;
         // Prevent re-enterability without calling path.skip.
-        if (t.isBlockStatement(path.parentPath) && t.isTryStatement(path.parentPath.parentPath))
+        if (path.parentPath.isBlockStatement() && path.parentPath.parentPath.isTryStatement())
```

Comment:
We had to change this because of:

```
> tsc && eslint . --ext .ts,.tsx,.js

src/debugTransform.ts:32:32 - error TS2345: Argument of type 'NodePath<Node>' is not assignable to parameter of type 'Node | null | undefined'.
  Property 'kind' is missing in type 'NodePath<Node>' but required in type 'TSPropertySignature'.

32         if (t.isBlockStatement(path.parentPath) && t.isTryStatement(path.parentPath.parentPath))
                                  ~~~~~~~~~~~~~~~

  node_modules/@babel/types/lib/index.d.ts:1160:5
    1160     kind: "get" | "set";
             ~~~~
    'kind' is declared here.

src/debugTransform.ts:32:69 - error TS2345: Argument of type 'NodePath<Node> | null' is not assignable to parameter of type 'Node | null | undefined'.
  Type 'NodePath<Node>' is not assignable to type 'Node | null | undefined'.

32         if (t.isBlockStatement(path.parentPath) && t.isTryStatement(path.parentPath.parentPath))
                                                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/debugTransform.ts:32


playwright-vscode on  update-babel [!] is 📦 v1.0.16 via  v18.17.1 took 4s
```

Note: Did not test this.